v2src/CMakeLists.txt

```diff
@@ -206,8 +215,10 @@ install(FILES "${AOTRITON_V2_BUILD_DIR}/${AOTRITON_LIBRARY_FILE}" DESTINATION ${
 
 # Python binding only available for AOTriton V2 API
 add_library(aotriton INTERFACE)
-add_dependencies(aotriton aotriton_v2)
+# add_dependencies(aotriton aotriton_v2)
 # target_link_libraries(aotriton INTERFACE ${CMAKE_INSTALL_PREFIX}/lib/libaotriton_v2.a)
 # target_include_directories(aotriton INTERFACE ${CMAKE_INSTALL_PREFIX}/include)
-target_link_libraries(aotriton INTERFACE ${AOTRITON_V2_BUILD_DIR}/${AOTRITON_LIBRARY_FILE})
-target_include_directories(aotriton INTERFACE ${CMAKE_CURRENT_SOURCE_PARENT_DIR}/include)
+# target_link_libraries(aotriton INTERFACE ${AOTRITON_V2_BUILD_DIR}/${AOTRITON_LIBRARY_FILE})
+# target_include_directories(aotriton INTERFACE ${CMAKE_CURRENT_SOURCE_PARENT_DIR}/include)
+# target_include_directories(aotriton INTERFACE ${CMAKE_CURRENT_SOURCE_PARENT_DIR}/include)
+target_link_libraries(aotriton INTERFACE aotriton_v2)
```

Comment:
@xinyazhang Are these changes related to the name suffix feature? Can you please explain this changeset?

Reply:
the suffix is not long added to the symbols, but also added to the output files. Linking `aotriton INTERFACE` can eliminate the necessary to construct the output file name with suffix again.